core/data/src/main/java/kr/co/cotton/core/data/news/repository/DefaultNewsRepository.kt

```diff
@@ -0,0 +1,30 @@
+package kr.co.cotton.core.data.news.repository
+
+import androidx.paging.Pager
```

Comment:
🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between